random/random.mbti

```diff
@@ -3,13 +3,19 @@ package moonbitlang/core/random
 // Values
 
 // Types and methods
-type RandomState
-impl RandomState {
-  gen_double(Self) -> Double
-  gen_int(Self) -> Int
-  gen_int64(Self) -> Int64
-  init_state(~seed : Int = ..) -> Self
-  new() -> Self
+type Rand
+impl Rand {
+  double64(Self) -> Double
+  int(Self) -> Int
+  int64(Self) -> Int64
+  int64N(Self, Int64) -> Int64
+  intN(Self, Int) -> Int
+  new(~seed : Bytes = ..) -> Self
```

Comment:
what does `intN` `N` mean here, can you give it a more meaningful name?

Reply:
`N` means range, thats what go's API looks like.

Probably rename to `int_range`?